Day-II-Calculator/calculator/src/components/ButtonComponents/NumberButton.js

```diff
@@ -1,2 +1,30 @@
 import React from 'react';
 import './Button.css';
+import WideButtons from './WideButton';
```

Comment:
Just a little thing. I would import this as `WideButton` without an "s" because the component is only technically one button (even though you are going to use it more than once) 

Kind of like how you did with `SquareButton`